src/smart-switch.js

```diff
@@ -85,13 +85,13 @@ class SmartSwitch extends BasePlugin {
     const responseTimeoutHandler = () => {
       this.logger.warn(`Timeout reached ${this.config.responseTimeoutSec} seconds, loading original source`);
     };
+
+    let url = SmartSwitchConfig.CDN_BALANCER_API_ENDPOINT.replace('{accountCode}', this.config.accountCode);
     const params = {
-      accountCode: this.config.accountCode,
-      originCode: this.config.originCode,
```

Comment:
so `originCode` is not in use? need to remove it from https://github.com/kaltura/playkit-js-smartswitch/blob/18cf01d75d3f01a3bf422c8f9904f4bb1ab27d72/src/smart-switch.js#L102

Reply:
it is included on the optional query params
https://documentation.npaw.com/npaw-integration/reference/get_-accountcode-decision
Checking with Gilad if it is mandatory from our side